client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -57,6 +57,7 @@ const onboarding: FlowV2 = {
 	async initialize() {
 		if ( await isMvpOnboardingExperiment() ) {
 			return stepsWithRequiredLogin( [
+				STEPS.UNIFIED_DOMAINS,
```

Comment:
We're adding domain selection back.

Reply:
We also need to add `STEPS.USE_MY_DOMAIN,` as the user might pick it in the domains step sidebar.

Also, if we keep the same steps, then we don't need the conditional? They're the same right now, bar the use my domain step.

Reply:
Sounds good, [added it here](https://github.com/Automattic/wp-calypso/pull/102912/commits/b7d03eec32d169f2d3e5596511adc1198564f3c8).